DatadogSDKBridge/Classes/Implementation/GlobalState.swift

```diff
@@ -0,0 +1,32 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2021 Datadog, Inc.
+ */
+
+import Foundation
+
+internal struct GlobalState {
+    /// Holds global attributes to be used with Logs and Traces, because they don't expose Global instances, unlike RUM
+    internal static var globalAttributes: [String: Encodable] = [:]
```

Comment:
Just checking - we don't need to make this thread-safe, as RN calls all APIs (logs / traces / rum) using a single `DispatchQueue` (think: on the same thread), right?

Reply:
speaking of React Native: yes, we have a single serial `DispatchQueue` for all modules. But bridge is supposed to be used not only with RN, so specifics for other platforms may be different, but we don't know this yet.

Reply:
This will trigger early if we use it simultaneously in other technology. As it's safe for RN, IMO it's good for now 👍.